src/main/java/org/jabref/gui/push/PushToEmacsSettings.java

```diff
@@ -11,18 +10,12 @@
 
 public class PushToEmacsSettings extends PushToApplicationSettings {
 
-    private final JTextField additionalParams = new JTextField(30);
-
-    @Override
-    public JPanel getSettingsPanel(int n) {
-        additionalParams.setText(Globals.prefs.get(JabRefPreferences.EMACS_ADDITIONAL_PARAMETERS));
-        return super.getSettingsPanel(n);
-    }
+    private final TextField additionalParams = new TextField();
 
     @Override
-    public GridPane getJFXSettingPane() {
+    public GridPane getJFXSettingPane(int n) {
         additionalParams.setText(Globals.prefs.get(JabRefPreferences.EMACS_ADDITIONAL_PARAMETERS));
-        return super.getJFXSettingPane();
```

Comment:
I would prefer if there is only one method that needs to be overriden to customize the settings pane.